### PR TITLE
Rename Venmo to VenmoPayments

### PR DESCRIPTION
**Pull request by sshropshire**
### Summary of changes

 - Rename `:Venmo` module to `:VenmoPayments`
 - Implement initial project structure for VenmoPayments module 

 ### Checklist

 ~- [ ] Added a changelog entry~

- @sshropshire

### CODE REVIEW

**Comment by sarahkoop** [comment]
Should we start a feature branch for all of the venmo work? I see we already have a `Venmo` module in `main`, but might be better for CHANGELOG clarity and to avoid accidental release to just move everything to a feature branch going forward

**Comment by sshropshire** [comment]
@sarahkoop changed the base to `venmo-feature` similar to iOS 👍

File: VenmoPayments/build.gradle [hunk 1]
```diff
@@ -26,7 +25,7 @@ android {
         targetCompatibility JavaVersion.VERSION_1_8
     }
     kotlinOptions {
-        jvmTarget = '1.8'
+        jvmTarget = JavaVersion.VERSION_1_8.toString()
```
Comment (sarahkoop) [inline VenmoPayments/build.gradle]: With new modules should we consider targeting Java 11 or 17 instead? Or do we want to keep all modules aligned in Java version?

File: VenmoPayments/build.gradle [hunk 2]
```diff
@@ -39,3 +38,10 @@ dependencies {
     androidTestImplementation deps.androidxJUnit
     androidTestImplementation deps.androidxEspressoCore
 }
+
+project.ext.name = "venmo-payments"
```
Comment (sarahkoop) [inline VenmoPayments/build.gradle]: Do the CI scripts for test / release already include running this module?

File: VenmoPayments/build.gradle (hunk 1 above)
Comment (sshropshire) [reply VenmoPayments/build.gradle]: I'd be down to go with 17 (or whatever BT v5 uses).

File: VenmoPayments/build.gradle (hunk 2 above)
Comment (sshropshire) [reply VenmoPayments/build.gradle]: Oh wow I'm not sure. We used to have to declare it explicitly in the build script 🤔

Comment (sshropshire) [reply VenmoPayments/build.gradle]: Removed in: [29597c1](https://github.com/paypal/paypal-android/pull/267/commits/29597c1366264ab2ec3d66e787f8c927daef111a)

File: VenmoPayments/build.gradle (hunk 1 above)
Comment (sshropshire) [reply VenmoPayments/build.gradle]: Went for Java 11: [5a0abbd](https://github.com/paypal/paypal-android/pull/267/commits/5a0abbd2f5e14134797c15119f34adebdf0f0f59). Java 17 was giving the CI build problems for some reason.

**Review by scannillo — APPROVED** [review]

**Review by richherrera — APPROVED** [review]

**Pull request merged**